scripts/ktlint-pre-commit

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
Instead of calling this ktlint-pre-commit, let's maybe call it ktlint-check ? or kotlin-lint-check? Want to remove the 'pre-commit' part out of it

Reply:
Done, resolving on behalf of Vinita as it's a small fix.